map/bookmark_manager.cpp

```diff
@@ -1893,6 +1893,8 @@ void BookmarkManager::LoadBookmarkRoutine(std::string const & filePath, bool isT
         else
           collection->emplace_back(std::move(fileSavePath), std::move(kmlData));
       }
+      else
+        base::DeleteFileX(fileSavePath);
```

Comment:
In my cases there was a problem with `fileSavePath` parsing and temp files were not cleaned up.

Reply:
Both if and else branches have this line. Put it before the if.

Reply:
What kind of problem did you have with parsing? Should it be fixed/handled properly?

Reply:
If input KML/KMZ file has invalid format we get `kmlData == nullptr`.